https://bugs.gentoo.org/686752

Description: Fix FTBFS with SoundTouch 2.1.2
 Explicitly disable exceptions in SoundTouch headers because Dolphin
 is compiled with -fno-exceptions.
Author: James Cowgill <jcowgill@debian.org>
Bug-Debian: https://bugs.debian.org/917671
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/Source/Core/AudioCommon/OpenALStream.h
+++ b/Source/Core/AudioCommon/OpenALStream.h
@@ -32,6 +32,7 @@
 #define BOOL SoundTouch_BOOL
 #endif
 
+#define ST_NO_EXCEPTION_HANDLING
 #include <soundtouch/SoundTouch.h>
 #include <soundtouch/STTypes.h>
 
